charts/kof-mothership/templates/kof-operator/operator-deployment.yaml

```
@@ -58,7 +58,8 @@ spec:
             value: "http://{{ .Release.Name }}-promxy:{{ .Values.promxy.service.servicePort }}/-/reload"
           - name: "RELEASE_NAMESPACE"
             value: {{ .Release.Namespace }}
-        image: "{{ .Values.kcm.kof.operator.image.repository }}:v{{ .Chart.Version }}"
+        image: >-
+          {{ .Values.kcm.kof.operator.image.repository }}:{{ default (printf "v%s" (default .Chart.Version .Chart.AppVersion))  .Values.kcm.kof.operator.image.tag }}
```

In `kof` `.Chart.Version == .Chart.AppVersion`, also `value1 | default value2` is simpler:
```suggestion
        image: {{ .Values.kcm.kof.operator.image.repository }}:{{ .Values.kcm.kof.operator.image.tag | default (printf "v%s" .Chart.Version) }}
```

---

Fixed in https://github.com/k0rdent/kof/pull/283#issuecomment-2992123375